tournament.py

```diff
@@ -84,18 +84,14 @@ def play_matches(cpu_agents, test_agents, num_matches):
     total_forfeits = 0.
     total_matches = 2 * num_matches * len(cpu_agents)
 
-    print("\n{:^9}{:^13}{:^13}{:^13}{:^13}{:^13}".format(
-        "Match #", "Opponent", test_agents[0].name, test_agents[1].name,
-        test_agents[2].name, test_agents[3].name))
-    print("{:^9}{:^13} {:^5}| {:^5} {:^5}| {:^5} {:^5}| {:^5} {:^5}| {:^5}"
+    print(("\n{:^9}{:^13}" + "{:^13}" * len(test_agents)).format(
```

Comment:
This is really awkward, but I admittedly just made a quick and dirty change here (and below) instead of refactoring the format strings.